backend/controllers/user_controller.go

```diff
@@ -29,10 +29,21 @@ type dataExtract struct {
 
 var (
 	//account_map   = make(map[string]string)
-	client_id     = os.Getenv("CLIENT_ID")
-	client_secret = os.Getenv("CLIENT_SECRET")
+	//client_id     = os.Getenv("CLIENT_ID")
+	//client_secret = os.Getenv("CLIENT_SECRET")
+	client_secret="27c5646e9c70d276c4b487fa9f5d17366f31f2b1"
+	client_id="cfa78f27867826309fb6"
```

Comment:
Secrets should never be hard coded in applications.

Reply:
These are something that should probably be read out of the `.env` file for local development. In production these would be set as environment variables at runtime. Also make sure the documentation is updated to add these parameters.